pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -317,15 +317,19 @@ func (o *LoginOptions) gatherProjectInfo() error {
 		}
 		o.Project = current.Name
 
-		fmt.Fprintf(o.Out, "You have access to the following projects and can switch between them with '%s project <projectname>':\n\n", o.CommandName)
-		for _, p := range projects.List() {
-			if o.Project == p {
-				fmt.Fprintf(o.Out, "  * %s\n", p)
-			} else {
-				fmt.Fprintf(o.Out, "    %s\n", p)
+		if len(projectsItems) > 50 {
+			fmt.Fprintf(o.Out, "You have access to %d projects, the list has been surpressed. You can list all projects with '%s projects'\n", len(projectsItems), o.CommandName)
```

Comment:
I think it'd be worth adding a comment before this line, explaining why we are doing this. That way we can prevent someone in the future from accidentally reverting this